src/main/java/roomescape/theme/service/ThemeService.java

```diff
@@ -44,6 +48,14 @@ public List<Theme> findAll() {
     }
 
     public List<Theme> findPopularThemes() {
-        return themeRepository.findTop10ThemesByReservationCountWithin7Days(POPULAR_THEME_DAYS, POPULAR_THEME_LIMIT);
+        final LocalDate sevenDaysAgo = LocalDate.now().minusDays(POPULAR_THEME_DAYS);
+        final LocalDate today = LocalDate.now();
```

Comment:
`LocalDate.now()`를 두번 호출할 필요 없이 아래와 같이 한번 호출한 뒤, 재사용하면 되지않을까요? 

```suggestion
        final LocalDate today = LocalDate.now();
        final LocalDate sevenDaysAgo = today.minusDays(POPULAR_THEME_DAYS);
```

Reply:
말씀해주신 대로 재사용하는 방법이 간결하기도 하고 시간에 대한 일관성을 지킬수 있을 것 같아요!